### PR TITLE
Added Pipfile and README to gae_dashboard

### PR DESCRIPTION
**Pull request by ishanip24**
## Summary:
Added README to gae_dashboard

Added Pipfile and Pipefile.lock to make setup easier for future.

Issue: INFRA-6453

## Test plan:
Proofread the readme, follow the commands for setup

### CODE REVIEW

File: gae_dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# gae_dashboard
+
+This folder contains various miscellaneous script that runs on Toby server.
+
+## Setup
+
+To run these scripts locally, you will need to install your local python dependencies using
```
Comment (dat-boris) [inline gae_dashboard/README.md:7]: Note: we might want to add a description to explain developer might need to install pipenv https://pipenv.pypa.io/en/stable/

**Review by dat-boris — COMMENTED** [review]
Congrats on the first PR on internal-webserver repo 🎉 

Looks good to me - but since I'm involved in the pairing, I will leave it to @latteier and @LoveAndCoding for the review :-) 

(and apology for not thinking about the pipenv installation guide during the pairing)

**Review by latteier — APPROVED** [review]
Thanks!

Might be worth mentioning the difference between the requirements.txt file and the Pipfile.

**Comment by dat-boris** [comment]
Hi @ishanip24 thanks for pushing the code!

I would suggest you keep https://github.com/Khan/internal-webserver/pull/7 and https://github.com/Khan/internal-webserver/pull/6 separate since they are not related to each other.  

In addition, https://github.com/Khan/internal-webserver/pull/6 is approved and ready to be merged, while https://github.com/Khan/internal-webserver/pull/7 still need to go through review.

If you need pairing to sort this out, happy to pair on this!

**Pull request merged**